kythe/go/extractors/config/extractor.go

```diff
@@ -51,6 +52,59 @@ func GitCopier(repoURI string) func(ctx context.Context, outputDir string) error
 	}
 }
 
+// LocalCopier returns a function that copys a local repository.
+func LocalCopier(repoPath string) func(ctx context.Context, outputDir string) error {
+	return func(ctx context.Context, outputDir string) error {
+		gitDir := filepath.Join(repoPath, ".git")
+		// TODO(danielmoy): consider extracting all or part of this
+		// to a more common place.
+		return filepath.Walk(repoPath, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if repoPath == path {
+				// Intentionally do nothing for base dir.
```

Comment:
Please mention in the doc comment that this assumes the output directory has already been created.